webui/src/lib/api/index.js

```diff
@@ -764,9 +794,14 @@ class Config {
         }
     }
     async getLakeFSVersion() {
-        const response = await apiRequest('/config/version', {
-            method: 'GET',
-        });
+        try {
+            const response = await apiRequest('/config/version', {
+                method: 'GET',
+            });
+        } catch (e) {
+            // avoid crush in case we are not logged in
+            return Promise.resolve({})
```

Comment:
Think we need to open an issue to fix this one - screens which doesn't have user authenticated / user logged in, should not render data using authenticated API. Also, it hides the fact that we failed and the empty object response is not the response from the server. Will be hard to debug later.

Reply:
opened https://github.com/treeverse/lakeFS/issues/3201, will add more details